core/web/presenters/ocr_keys.go

```diff
@@ -57,9 +58,9 @@ func NewOCR2KeysBundleResource(key ocr2key.KeyBundle) *OCR2KeysBundleResource {
 	return &OCR2KeysBundleResource{
 		JAID:              NewJAID(key.ID()),
 		ChainType:         string(key.ChainType()),
-		OnchainPublicKey:  key.OnChainPublicKey(),
-		OffChainPublicKey: hex.EncodeToString(key.OffchainPublicKey()),
-		ConfigPublicKey:   hex.EncodeToString(configPublic[:]),
+		OnchainPublicKey:  fmt.Sprintf("ocr2on_%s_%s", key.ChainType(), key.OnChainPublicKey()),
+		OffChainPublicKey: fmt.Sprintf("ocr2off_%s_%s", key.ChainType(), hex.EncodeToString(key.OffchainPublicKey())),
```

Comment:
nit: We should consistently use `Offchain` and `Onchain` everywhere, _not_ `OffChain` and `OnChain`

Reply:
oh jeeze looking over the codebase it's very 50/50 split :laughing:. Let's push that to a follow up PR though.

Reply:
Sounds good